resources/views/dashboard.blade.php

```diff
@@ -1,17 +0,0 @@
-<x-app-layout>
-    <x-slot name="header">
```

Comment:
You should keep this dashboard blade and render summarized information like counts, total, and specific information which are needed to render

Reply:
Yeah got ill take revert it

Reply:
Reverted all changes done to Dashboard... create a new issue to properly implement the dashboard